views/publication/details.templ

```diff
@@ -226,18 +226,22 @@ templ DetailsBody(c *ctx.Ctx, p *models.Publication) {
 					@display.Field(display.FieldArgs{
 						Label: c.Loc.Get("builder.has_confidential_data"),
 						Value: c.Loc.Get("confirmations." + p.HasConfidentialData),
+						Required: true,
 					})
 					@display.Field(display.FieldArgs{
 						Label: c.Loc.Get("builder.has_patent_application"),
 						Value: c.Loc.Get("confirmations." + p.HasPatentApplication),
+						Required: true,
 					})
 					@display.Field(display.FieldArgs{
 						Label: c.Loc.Get("builder.has_publications_planned"),
 						Value: c.Loc.Get("confirmations." + p.HasPublicationsPlanned),
+						Required: true,
 					})
 					@display.Field(display.FieldArgs{
 						Label: c.Loc.Get("builder.has_published_material"),
 						Value: c.Loc.Get("confirmations." + p.HasPublishedMaterial),
+						Required: true,
```

Comment:
I assume it was decided to just add the "Required" label but don't enforce it? Otherwise, I'm missing validation logic for this.

Reply:
Yes, we're not enforcing it, but mostly getting people's attention to at least say "I don't know" if they don't.
I am working on a better version of this with Marthe, but in the meantime to support the current reviewers workflow we added this.